prow/prstatus/prstatus.go

```diff
@@ -284,6 +301,31 @@ func (da *DashboardAgent) QueryPullRequests(ctx context.Context, ghc githubClien
 	return prs, nil
 }
 
+func (da *DashboardAgent) HeadContexts(ghc githubClient, pr PullRequest) ([]Context, error) {
+	org := string(pr.Repository.Owner.Login)
+	repo := string(pr.Repository.Name)
+	combined, err := ghc.GetCombinedStatus(org, repo, string(pr.HeadRefOID))
```

Comment:
I think this is fine for this PR, but cc @cjwagner IIRC we have to do some other tricks in tide to reliably get the statuses